src/services/organizationService.ts

```diff
@@ -140,6 +141,97 @@ function mergePublicAdministrationsAndOrganizations(
   );
 }
 
+/**
+ * Given an organization instance, forcefully deletes from the database
+ * the organization and its legal representative
+ * @param organizationInstance The instance of the organization to be deleted.
+ */
+export async function deleteOrganization(
```

Comment:
why we need this ? can we a soft delete instead ?

Reply:
If we use a soft delete here, then we should change the logic of the organization creation, because we should handle the case of registering again a previously deleted organization. Indeed, this method should be used only in order to cancel a registration process in a pre-draft status: when in such status, both the organization and its legal representative can be safely hardly deleted because their existence has no value outside the context of their registration process.

Reply:
can we add this as a comment above the function ?